challenge-react/src/helpers/donation.js

```diff
@@ -0,0 +1 @@
+export const summaryDonations = (donations) => donations.reduce((accumulator, value) => accumulator + value)
```

Comment:
Reduce with no initial value makes this function will receive an empty array. It will return an error.